### ISSUE TITLE
become an Enspiral Venture

### ISSUE THREAD

**Issue by whit537** [gratipay/inside.gratipay.com#421]
> Enspiral is a collective of individuals and ventures, this document outlines how this works for individual ventures and how they relate to other parts of the network.
> 
> Enspiral is not a traditional incubator or accelerator. Enspiral does not have any dedicated staff to support Enspiral Ventures. We are a collective of people and companies who support each other to create a thriving society.

http://static1.squarespace.com/static/5169f4a1e4b0fdc6c23ef665/t/535e0ecfe4b0ad70e229b48b/1398673103417/Enspiral_ventures+.pdf

**Comment by whit537** [comment]
> #### Becoming an Enspiral Venture
>
##### Process
>
1. New company writes a company introduction
2. Their introduction is posted in the members Loomio, to let members know about them and
provide a space for discussion, questions, comments, etc (no proposal yet).
3. A one month “getting to know” you phase starts, which goes both ways of them getting
to know us and us getting to know them. New company people come on G+ and Loomio,
members make an effort to talk to them, meet for coffee, etc, and they ask any questions and
clarify the value proposition and values fit on both sides.
4. We ask the company if they definitely want to join, and if so, a proposal is started in the above
Loomio discussion for members to officially approve or not.
5. If approved (by consensus - with 75% member engagement), the company and the Foundation
officially determine the contribution relationship (revenue share, etc).
6. We have a welcome celebration!

**Comment by whit537** [comment]
Bringing over from https://github.com/gratipay/inside.gratipay.com/issues/72#issuecomment-159727340:

 - http://p2pfoundation.net/Open_Cooperatives#The_Enspiral_Network
 - "[Inventing a New Organizational Operating System](https://www.youtube.com/watch?v=4ABqCgKi4X0)"
 - "[No Boss Does Not Mean No Leadership](https://medium.com/enspiral-tales/no-boss-does-not-mean-no-leadership-c4c97c660252)"
 - "[How to Grow Distributed Leadership](https://medium.com/enspiral-tales/how-to-grow-distributed-leadership-7f6b25f0361c)"

**Comment by whit537** [comment]
I just finished watching "[Inventing a New Organizational Operating System]([Inventing a New Organizational Operating System](https://www.youtube.com/watch?v=4ABqCgKi4X0))," and that's what inspired me to make this ticket, along with starting to wade into the Enspiral/Loomio community (https://github.com/loomio/loomio/pull/2755, https://github.com/loomio/tech-manual/pull/2, [loomio intro](https://www.loomio.org/d/LuAlGpC3/introduce-yourself-to-the-loomio-community?comment=859614)). I am attracted to the spirit and the vibrancy of Enspiral/Loomio. They seem to me to be a welcoming, healthy community. And they're focused on work! They get stuff done! :dancer: 

I think that Gratipay may have something to offer the Enspiral community: an open payments venture. In Loomio they're hacking together payments using Chargify (cf. https://github.com/loomio/loomio/pull/2755#issuecomment-159741662). I think Gratipay could evolve into a payments partner for other ventures—one that is open and aligned.

If Gratipay is *not* something that Enspiral needs/wants, then presumably it's because they have their own answer to the problems we're trying to solve. And if *that's* the case, then it may make sense for us to quit and join them anyway. :-)

**Comment by whit537** [comment]
cc: @clone1018 @tshepang @webmaven @mattbk @kzisme @kaguillera @rohitpaulk et al. Pretty big convo here, I think. :-)

Is this something we're interested in pursuing?

**Comment by gdpelican** [comment]
Hey @whit537 this sounds like a really interesting idea. Feel free to reach out with questions / concerns as they arise, and let's chat if it seems like there's some traction here.

**Comment by whit537** [comment]
Thanks @gdpelican! Will do! :-)

**Comment by ahdinosaur** [comment]
hey @whit537, this is exciting. :) i think an open payments venture in Enspiral would be epic. /cc @jessykate

in parallel to this conversation about *Gratipay joining as an [Enspiral venture](https://github.com/enspiral/agreements/blob/master/agreements/ventures.md)*, @whit537 do you want to *personally join as [an Enspiral contributor](https://github.com/enspiral/agreements/blob/master/agreements/people.md#contributors)*? if so, i'd be happy to invite you, all i need from you is an email with a short "bio" (what do you want other people in Enspiral to know about you?). then if you're really serious, i recommend going to an [Enspiral retreat](http://retreat.enspiral.com/) which happens twice a year. :cat:

**Comment by ahdinosaur** [comment]
oh, also folks here might be interested in [how money works in Enspiral](https://github.com/enspiral/guides/blob/master/money.md). :money_with_wings:

**Comment by whit537** [comment]
Thanks for the kind and generous offer, @ahdinosaur! I'm diving into all of the Enspiral docs. There's a lot to digest! :-)

I have some questions gradually forming over here, will follow up when I'm able ...

**Comment by jessykate** [comment]
super cool guys! thanks for the tag @ahdinosaur :). i'm really keen about a shared payments layer, as is @jsayles. i'd be particularly interested if there is a convo to be had about ACH payment infrastructure. don't want to derail this ticket so feel free to ping me separately if desired. really cool poking around and learning about gratipay! :).

**Comment by whit537** [comment]
Thanks for chiming in @jessykate! :-)

We came pretty close earlier this year to landing a partnership with a bank to get on the ACH network. It fell through because our risk controls weren't far enough along. Details on https://github.com/gratipay/gratipay.com/issues/3366. I guess the good news is that we now know a lot more about what it will take. :-) Our weekly Radar ticket has [a high-level roadmap](https://github.com/gratipay/inside.gratipay.com/issues/425#issuecomment-160823625). ACH payments fall under "direct payouts" on the diagram there.

**Comment by jsayles** [comment]
This is very interesting for sure.

**Comment by whit537** [comment]
@jessykate @jsayles What's your background with ACH and/or payments? What are you looking to build? :-)

**Comment by jsayles** [comment]
I run a coworking space and handle the billing, IT, and systems integration.  We use ACH as well as credit cards.  We've talked about a system that would save on merchant service fees and also empower members to vote with those saved dollars while simultaneously showing their loyalties.  Much like how Brown Paper Tickets allows you to pick which charities should benefit from the ticket processing fees they collect.

**Comment by whit537** [comment]
@jsayles Sounds like you and @elliottw are in the same boat. He runs [Catapult](http://catapultpgh.org/) here in Pittsburgh, and accepts pay-what-you-want dues [via Gratipay](https://gratipay.com/catapultpgh/) (in addition to regular membership fees via Memberful). Gratipay itself doesn't charge hard fees (we're pay-what-you-want [on our own platform](https://gratipay.com/Gratipay/)), though we haven't approached it from the savings-to-charity angle yet. We do pass through credit card fees at cost. I think probably [Dwolla](https://www.dwolla.com/) is the best bet for fee-less money transfer. We've *just* started adding them as a payout option over on https://github.com/gratipay/gratipay.com/issues/726; the payin side is https://github.com/gratipay/gratipay.com/issues/65. Have you used Dwolla before?

**Comment by whit537** [comment]
@gdpelican and I have scheduled an open call for December 29 at 2pm UTC, to discuss https://github.com/loomio/loomio/issues/2814#issuecomment-165787594 as well as this ticket.

**Comment by whit537** [comment]
Call: https://www.youtube.com/watch?v=MmksnIsEJPk.

**Comment by whit537** [comment]
Thanks @gdpelican. :-)

**Comment by whit537** [comment]
A second misfire at https://github.com/loomio/loomio/issues/2818#issuecomment-184426094. I am not getting good vibes from Loomio as an open-source project. :disappointed:

**Comment by rdbartlett** [comment]
Hey @whit537 I care an awful lot about creating good vibes for people contributing to Loomio. 

We're a startup that's trying to balance the needs of a tight crew of developers in the room together here in NZ, with the needs of a world of potential open source contributors. 

We've been super internally focussed for a while, and we're now in the process of turning our attention outward, e.g. agreeing a design and contribution process that suits the needs of the project.

I'd love for you to check back in after a few months and give me another vibe rating.

**Comment by whit537** [comment]
Thanks for reaching out, @rdbartlett. You've got a big challenge ahead of you!

I'm doing some contract work for the [Center for Open Science](http://cos.io/) (COS), and your situation reminds me of theirs. They're a three or four year old startup that, like Loomio, has met with some success. While they aspire to openness and they do have most of their code in [public repos](https://github.com/CenterForOpenScience) on GitHub, their team of ~50 is *all* on-site (I'm the first remote), and their company culture is centered on face-to-face meetings, private chat (Flowdock in their case, ~= Slack), and private Jira. I don't believe it's happened yet for them, but if an unassuming outsider tried to start contributing as if it were any old open-source project, I imagine COS's organizational response would be similar to Loomio's ... which reminds me of a Far Side classic. :-)

![How nature says, "Do not touch."](https://cloud.githubusercontent.com/assets/134455/13112818/795c39a2-d55a-11e5-90b4-4f90fa4edd54.jpg)

In other words, openness is more than just public repos on GitHub, it's about organizational culture—and that gets harder and harder to change the bigger you get.

On possible point of convergence: I'm planning to be at OuiShare Fest in May (#314), and @francesca-sp has expressed interest in having "a group session about open source / open company projects" (that's from a private email reiterating a sentiment she also [expressed](https://www.youtube.com/watch?v=dcaMkwFhyV0#t=54m05s) on a recent video call). Are you planning to be at OuiShare Fest, @rdbartlett? I'm sure Loomio and Enspiral will be represented, at the least. Perhaps that'll be a chance to move this conversation forward?

**Comment by rdbartlett** [comment]
There will definitely be Enspiral folk there, but not me - hopefully next time though!

**Comment by whit537** [comment]
Small world moment over the weekend ... I hung out with someone living here in [Ambridge](https://gratipay.news/meet-ambridge-8b9b2b6f8628#.9nzabvjqw), who turns out to be from New Zealand. The conversation went something link this:

"What do you do?"
"I'm working on a startup called Gratipay."
"Oh yeah? I have some friends back in New Zealand working on a startup."
"What's it called?"
/me waits for it
"Loomio."
"Yeah, I'm off to a bit of a rough start with the Loomio folks, to be honest."
"Oh no! They're all really nice! You should just go to New Zealand and meet them."
"Yeah, that's the problem."

:stuck_out_tongue_winking_eye: 

She mentioned you, @rdbartlett, as well as @mhjb. :-)

P.S. GitLab has been putting out [great material lately](https://about.gitlab.com/2016/03/23/remote-communication/) on how to run an open company. Worth checking out if you're serious about evolving Loomio's culture.

**Comment by rdbartlett** [comment]
New plan: I'm coming to OuiShare fest. Let's catch up Chad.

**Comment by mhjb** [comment]
Hi Chad — I'd love to smooth out that rough start; perhaps you could ask your friend to get in touch (matthew@loomio.org) if that seemed appropriate? 

Loved the GitLab article.

**Comment by whit537** [comment]
@rdbartlett Ooooooh yikes. Please tell me that this thread is not the main reason you're doing that. :flushed: 

@mhjb Happy to share her name with you in email, not sure we need to drag her in further. Honestly I told here that in my view Loomio/Enspiral and Gratipay have years ahead of us in this new collaborative economy scene that's just beginning to emerge. I am quite hopeful that we will be fast friends in time! :-)

**Comment by rdbartlett** [comment]
Hehehe Chad I'm happy to say this thread had no bearing on my decision to
fly to Europe! Looking forward to fast-friending!

**Comment by whit537** [comment]
> I'm happy to say this thread had no bearing on my decision to fly to Europe!

Phew! :relieved:

> Looking forward to fast-friending!

Likewise! See you in Paris! :smile: :fr:

**Comment by whit537** [comment]
Alright, so it's now been over two months, and I haven't circled back around here yet since Paris. @rdbartlett and I were indeed able to catch up at OuiShare Fest (#314). We spent 45 minutes in the burrito line together, getting to know one another and our respective organizations (before running into each other on the dance floor later that night ;). The key take-away from the conversation for me was @rdbartlett's insight, "Tension doesn't mean no, it means not yet." I keep looking for more to say here by way of expansion, but I think it's best just to leave that as the word for now.

@rdbartlett It was great to be with you and all of the other Enspiralites at OuiShare Fest, especially @derekrazo (yay Measuring and Distributing Value! :) and @genevievebelle (yay Lille! :). I look forward to Gratipay and Enspiral's next semi-serendipitous encounter! ❤️  ☺️

**Comment by rdbartlett** [comment]
I've been meaning to circle back to this thread too, thanks Chad.

One other takeaway from me was about the tension of design-vs-democracy in open source software products. I was coming from the perspective that, since Loomio is a consumer-facing app that prioritises accessibility over any other criteria, we basically need dictatorial oversight over the UX and UI design. I had imagined that would be one person's job. After the burrito conversation, I felt more open to the possibility that this kind of oversight could come from agreed protocols, rather than a designated dictator.

Also it was so great having dancefloor time with you matey!! 💃
